collect_app/src/androidTest/java/org/odk/collect/android/support/rules/RetryOnDeviceErrorRule.kt

```diff
@@ -14,10 +14,10 @@ class RetryOnDeviceErrorRule : TestRule {
                     base.evaluate()
                 } catch (e: Exception) {
                     if (e is PerformException) {
-                        Timber.e("RetryOnDeviceErrorRule: Retrying due to PerformException!")
+                        Timber.e(Error("RetryOnDeviceErrorRule: Retrying due to PerformException!"))
```

Comment:
These should all be warnings instead of errors as they are test logging. Let's change them while we're here.

Reply:
Fixed.